### PR TITLE
Added translation adjustments pt_PT

### PR DESCRIPTION
**Pull request by adrianomarinho**

### CODE REVIEW

File: locale/pt_PT.utf8/LC_MESSAGES/rosariosis.po [hunk 1]
```diff
@@ -163,13 +163,13 @@ msgstr "OK"
 
 #: index.php:86
 msgid "Your account has not yet been activated."
-msgstr ""
+msgstr "Sua conta ainda não foi ativada."
 
 #: index.php:86
 #: index.php:151
 #, fuzzy
```
Comment (francoisjacquet) [inline locale/pt_PT.utf8/LC_MESSAGES/rosariosis.po]: The `fuzzy` mentions a Fuzzy translation (the original translation is generally outdated due to changes in the string).

Comment (francoisjacquet) [inline locale/pt_PT.utf8/LC_MESSAGES/rosariosis.po]: Here, the `fuzzy` mention is still present, though you have translated / updated the translation.

File: locale/pt_PT.utf8/LC_MESSAGES/rosariosis.po [hunk 2]
```diff
@@ -163,13 +163,13 @@ msgstr "OK"
...
 msgid "You will be notified when it has been verified by a school administrator."
-msgstr "Sua conta foi criada. Você será notificado por e-mail quando é verificada pela administração da escola e você pode log in"
+msgstr "Sua conta foi criada. Você será notificado por e-mail e quando for verificado pela administração da escola você pode efetuar o login"
```
Comment (francoisjacquet) [inline locale/pt_PT.utf8/LC_MESSAGES/rosariosis.po]: Here, the `fuzzy` mention is still present, though you have translated / updated the translation.

**Comment by francoisjacquet** [comment]
Thank you for your contribution @adrianomarinho 

Looking at your modifications, you did not remove the Fuzzy states of the strings you translated (see for example: https://github.com/francoisjacquet/rosariosis/pull/129/files#diff-3d7fc628b0fdf787bdb7bd23d6b498dcR170).
Did you actually use [Poedit](http://poedit.net/)?

_Obrigado_

**Comment by adrianomarinho** [comment]
Good afternoon.

I'm sorry for the late reply. I ended up the wrong file, however, have
already sent the file corrected by Poedit.

Big hug.

Regards,
Adriano Marinho

Em qui, 25 de fev de 2016 às 19:47, François Jacquet <
notifications@github.com> escreveu:

**Pull request merged**